Objects/longobject.c

```diff
@@ -4554,7 +4554,7 @@ long_or(PyObject *a, PyObject *b)
 }
 
 static PyObject *
-long_long(PyObject *v)
+long_long(PyObject *v, PyObject *Py_UNUSED(ignored))
```

Comment:
It is used both as `PyCFunction` and `getter`. It would be safer to declare the second parameter as `void *`.

Reply:
Done.